frontend/public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Your React App</title>
```

Comment:
Probably could do with being a bit more descriptive
```suggestion
  <title>XChem Lab</title>
```